examples/Benchmark.py

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Oct 18 15:53:27 2016
+
+@author: Michael Wu
+
+Benchmark test
+Giving performances of RF(scikit) and MultitaskDNN(TF)
+on datasets: muv, nci, pcba, tox21, toxcast, sider
+
+time estimation(on a nvidia tesla K20 GPU):
+tox21 - dataloading: 30s
+      - tf: 40s
+muv   - dataloading: 400s
+      - tf: 250s
+pcba  - dataloading: 30min
+      - tf: 2h
+sider - dataloading: 10s
+      - tf: 60s
+toxcast dataloading: 70s
+	tf: 70s
+(will include more)
```

Comment:
Could you add a total-time-to-run number at the bottom? (Just so that users can get a quick sense of how long the full benchmark suite should take).